src/main/resources/data.sql

```diff
@@ -2,14 +2,14 @@ INSERT INTO room(name) VALUES ('Phelps 2510');
 INSERT INTO room(name) VALUES('Phelps 3526');
 INSERT INTO room(name) VALUES('CSIL');
 
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Monday', 1700, 1900,1);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Monday', 1900, 2100,2);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Tuesday', 1900, 2100,2);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Wednesday', 1700, 2100,1);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Thursday', 1900, 2100,2);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Friday', 1000, 1200,3);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Friday', 1300, 1500,2);
-INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Friday', 1500, 1700,1);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Monday', 1700, 1900,1);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Monday', 1900, 2100,2);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Tuesday', 1900, 2100,2);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Wednesday', 1700, 2100,1);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Thursday', 1900, 2100,2);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Friday', 1000, 1200,3);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Friday', 1300, 1500,2);
+-- INSERT INTO room_availability(quarter, day, start_time, end_time, room_id) VALUES('W20', 'Friday', 1500, 1700,1);
```

Comment:
Can we delete this instead of commenting it out?

Reply:
Fixed.